src/xrpld/rpc/handlers/AccountTx.cpp

```diff
@@ -429,7 +431,13 @@ doAccountTxJson(RPC::JsonContext& context)
         return RPC::invalid_field_error(jss::forward);
     }
 
-    args.limit = params.isMember(jss::limit) ? params[jss::limit].asUInt() : 0;
+    if (auto const err =
+            RPC::readLimitField(args.limit, RPC::Tuning::accountTx, context))
+        return *err;
+
+    if (args.limit == 0)
+        return rpcError(rpcINVALID_PARAMS);
```

Comment:
```suggestion
    auto const& jvLimit = params[jss::limit];
    if (jvLimit.asUInt() == 0)
        return rpcError(rpcINVALID_PARAMS);
```

This should work, since the `RPC::readLimitField` will set `args.limit` to the minimum value of the `accountTx`'s `LimitRange`. You wouldn't need to check that it's a uint anymore, since the `readLimitField` already has done so.

Reply:
I was thinking about adding it directly into `readLimitField`, but unsure if that'll break something in a separate RPC.

Reply:
Give it a shot, I'd say - assuming test coverage is sufficient.

Reply:
ce4d641b28711c37a5e48aa85a318ce2b057ebc8

Reply:
Since this does change the error response for at least one RPC method, I'd like @godexsoft to take a second look.